### PR TITLE
Added Spanish translation for example check image and Company types

### PR DESCRIPTION
**Pull request by PrashantMangukiya**
@iwiznia  PR is ready for review.

### Details
Spanish translation was missing for Company types, and Routing and Bank Account Number example image in VBA flow. So added translation for both.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5992

### Tests | QA Steps
1. Set app to spanish
2. Navigate to bank account flow > Manually set bank account
3. Check the image example above the routing and bank account number
4. Enter test credentials and proceed to company step
5. Click on the company type dropdown

QA Check 1:  Now example check image shows in Spanish Or English as per language preference.
QA Check 2:  Company Types dropdown will be translated as per language preference.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
https://user-images.githubusercontent.com/7823358/140092975-6e9d8045-940b-4aba-8e99-664ed494c3c6.mp4

#### Mobile Web
https://user-images.githubusercontent.com/7823358/140093036-f8aa2e90-ab21-41b9-b4e7-b8e8b8c1f03d.mp4

#### Desktop
https://user-images.githubusercontent.com/7823358/140093103-3bc97e98-59c3-4514-b726-d5b93f10bdb7.mp4

#### iOS
https://user-images.githubusercontent.com/7823358/140093177-584894ce-94c5-48b4-b02a-8743c43899c4.mp4

#### Android
https://user-images.githubusercontent.com/7823358/140093255-33d290b4-4613-4545-a112-7183c55d4d88.mp4

### CODE REVIEW

File: src/languages/es.js [hunk 1]
```diff
@@ -586,6 +586,14 @@ export default {
         listOfRestrictedBusinesses: 'lista de negocios restringidos',
         incorporationDatePlaceholder: 'Fecha de inicio (aaaa-mm-dd)',
         companyPhonePlaceholder: '(prefijo) + (número)',
+        incorporationTypes: {
+            LLC: 'LLC',
+            CORPORATION: 'Corp',
+            PARTNERSHIP: 'Camaradería',
```
Comment (iwiznia) [inline src/languages/es.js:592]: This does not sound like the correct word, I have no idea what this partnership type of company is or how is it called in spanish... let me post this on slack and get back to you

Comment (iwiznia) [reply src/languages/es.js:592]: Let's use `Sociedad`

Comment (PrashantMangukiya) [reply src/languages/es.js]: Corrected and pushed new commit. Thanks.

**Comment by PrashantMangukiya** [comment]
@iwiznia   It looks like something wrong with E2E automation, as E2E test is failing after 9 minutes. Here is screenshot: 

Screenshot 2021-11-03 at 11 45 34 PM

**Comment by iwiznia** [comment]
That's odd, can you try:
- Merging main into this branch
- Restarting the test
?

**Review by iwiznia — APPROVED** [review]

**Comment by PrashantMangukiya** [comment]
It is not showing me any option to restart the test etc. I think someone from review team may have that permission.

**Review by timszot — APPROVED** [review]
Restarted the tests, will wait to and see what happens.

**Comment by ogumen** [comment]
The accessibility issues found in this PR:
**Web**
1. I assume that not all parts of the app are being translated to Spanish once changing the language in Preferences, the ones that are translated should have lang="es" applied - failure of WCAG SC 3.1.2. Once the entire page is being translated to Spanish - the `<html lang="en">` should have "es" value instead (it will be treated under WCAG SC 3.1.1).
Language of page
Example of language of parts

**Pull request merged**